diff -urN libetpan-0.54/configure.ac libetpan-0.54.new/configure.ac
--- libetpan-0.54/configure.ac	2008-02-20 23:40:31.000000000 +0100
+++ libetpan-0.54.new/configure.ac	2008-07-17 20:03:41.000000000 +0200
@@ -376,7 +376,7 @@
   case "$DBVERS" in
     3)
     AC_MSG_RESULT([version 3.x or above])
-    for lib in db-4.5 db-4.4 db-4.3 db-4.2 db-4.1 db-4.0 db-4 db4 db-3.2 db-3 db3 db; do
+    for lib in db-4.7 db-4.6 db-4.5 db-4.4 db-4.3 db-4.2 db-4.1 db-4.0 db-4 db4 db-3.2 db-3 db3 db; do
       if test "x$DBLINKED" = "x0"; then
 dnl        AC_CHECK_LIB($lib, db_create, [DBLIB="-l$lib"; DBLINKED=1], [])
 dnl installations of libdb4 function names are defined in db.h
