--- libetpan-1.7.2/configure.ac~	2016-05-26 07:27:47.000000000 +0200
+++ libetpan-1.7.2/configure.ac	2016-08-05 14:44:19.882952496 +0200
@@ -255,7 +255,7 @@
 if test "x$with_gnutls" = "xyes"; then
   AC_DEFINE([USE_GNUTLS],1, [Define to use GnuTLS])
   AC_DEFINE([USE_SSL], 1, [Define to use OpenSSL])
-  GNUTLSLIB="-lgnutls -lgcrypt -lgpg-error -lz"
+  GNUTLSLIB="-lgnutls"
 else
   GNUTLSLIB=""
 fi
@@ -618,12 +618,12 @@ else
       AC_CHECK_HEADER(curl/curl.h, [libcurl=yes])
       if test "x$libcurl" = "xyes"; then
           AC_MSG_CHECKING([whether curl-config hints compiles and links fine])
-          OLDFLAGS="$LDFLAGS"
-          LDFLAGS="$LDFLAGS `$libcurlconfig --libs 2>/dev/null`"
+          OLDLIBS="$LIBS"
+          LIBS="$LIBS `$libcurlconfig --libs 2>/dev/null`"
           AC_TRY_LINK([], [curl_easy_init();], [libcurl=yes], [libcurl=no])
           AC_MSG_RESULT([$libcurl])
           if test "x$libcurl" = "xno"; then
-              LDFLAGS="$OLDFLAGS"
+              LIBS="$OLDLIBS"
           fi
       fi
       
@@ -673,12 +673,12 @@ else
   AC_CHECK_HEADER(expat.h, [libexpat=yes])
   if test "x$libexpat" = "xyes"; then
     AC_MSG_CHECKING([whether libexpat compiles and links fine])
-    OLDFLAGS="$LDFLAGS"
-    LDFLAGS="$LDFLAGS -lexpat"
+    OLDLIBS="$LIBS"
+    LIBS="$LIBS -lexpat"
     AC_TRY_LINK([], [XML_SetElementHandler();], [libexpat=yes], [libexpat=no])
     AC_MSG_RESULT([$libexpat])
     if test "x$libexpat" = "xno"; then
-      LDFLAGS="$OLDFLAGS"
+      LIBS="$OLDLIBS"
     fi
   fi
 fi
